public_budget,sipreco_purchase,sipreco_subsidy_management: Serveral F…

iga-adhoc opened the pull request:
…ixes

iga-adhoc committed:
public_budget,sipreco_purchase,sipreco_subsidy_management: Serveral Fixes

roboadhoc commented:
[![Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/odoo-public-administration/pull/573.png)](https://mergebot.adhoc.ar/ingadhoc/odoo-public-administration/pull/573)

jjscarafia committed:
[FIX] tax settlement

jjscarafia committed:
[REM] deprecate withholding return

1. no lo venian usando y tenia errores
2. si se requiere implementar en 18 el approach sería completamente distinto

jjscarafia committed:
[FIX] payment states usage

1. posted no longer exists, now is "in_proccess" or "paid"
2. "to pay" amount should be comuted like this sipreco_state not in ['draft', 'canceled']

jjscarafia committed:
[FIX] advance transactions amounts

Para que el domnio no quede complejo convertimos sipreco_state a computado total es un campo muy livian
En un commit separado vamos a ver de unificar en utilizar sipreco_state en vez de algunas veces state y otras approval_state
Aprovechamos a cambiar para agrupar por sipreco state en vez de state

jjscarafia committed:
[REF] cleanup and accounts domain fixes

jjscarafia committed:
[FIX] advance transactions

Mejoramos la implementación
Además ahora funciona bien crear facturas en transacciones de adelanto con el "mass invoices" y el uno a uno, antes solo funcionaba con una a una..

jjscarafia commented:
@roboadhoc r+

roboadhoc commented:
@iga-adhoc @jjscarafia because this PR has multiple commits, I need to know how to merge it:

* `merge` to merge directly, using the PR as merge commit message
* `rebase-merge` to rebase and merge, using the PR as merge commit message
* `rebase-ff` to rebase and fast-forward

jjscarafia commented:
@roboadhoc rebase-ff

roboadhoc commented:
Merge method set to rebase and fast-forward.
